src/routes/auth/login/LoginCallback.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import queryString from 'query-string';
+import { useAuth } from 'contexts/authContext';
+
+import '../styles.scss';
+
+const LoginCallback = ({ history }) => {
+  const { saveLinkedInToken } = useAuth();
+  const { code } = queryString.parse(window.location.search);
+  if (code) {
+    saveLinkedInToken({ code });
+    history.push('/login');
```

Comment:
Don't think we need to push, the login function will automatically refresh (if AuthService is used)